### ISSUE TITLE
Repeated and missed id of hyperpartisan dataset

### ISSUE THREAD

**Issue by tsupei** [allenai/longformer#181]
I am trying to compare my model with longformer, however, I found there are some repeated ids both in test and train dataset. 
For example, 517, 24, 459, ... show in both test and train splits. Also, there are also disappeared ids such as 9 and 13. According to #15, for hyperpartisan dataset you use 80/10/10 splits, it seems that there should be no repeated ids in different splits?

**Comment by armancohan** [comment]
Yes, there seems to be some duplicates in the splits. We inherited these splits from another internal project and realized this issue later. Note that these splits are on the "training" set of the original dataset and we don't use the test set.
For reproducibility, we released the exact splits that we used; as long as you use the same data splits and preprocessing script as ours your results should be comparable.
